modules/ui/bin/jenkins-build-deploy.sh

```diff
@@ -23,6 +23,8 @@
 #   nexus_repository_id          : nexus milestone repository id
 #   nexus_snapshot_repository_id : nexus snapshot repository id
 
+#This file assumes that this script is being executed from the root directory of project. E.g. .../wasabi/
```

Comment:
Is the update to 'modules/ui/bin/jenkins-build-deploy.sh' generic or specific to the atomic experiment creation feature?